python/lsst/meas/extensions/shapeHSM/__init__.py

```diff
@@ -39,7 +39,7 @@
                     Control=HsmShapeRegaussControl, executionOrder=BasePlugin.SHAPE_ORDER)
 wrapSimpleAlgorithm(HsmSourceMomentsAlgorithm, name="ext_shapeHSM_HsmSourceMoments",
                     Control=HsmSourceMomentsControl, executionOrder=BasePlugin.SHAPE_ORDER)
-wrapSimpleAlgorithm(HsmSourceMomentsAlgorithm, name="ext_shapeHSM_HsmSourceMomentsRound",
+wrapSimpleAlgorithm(HsmSourceMomentsRoundAlgorithm, name="ext_shapeHSM_HsmSourceMomentsRound",
```

Comment:
Oh, that's definitely a bug! Good catch.

Reply:
FYI: It wasn't a bug, since whether round moments are calculated or not, was entiredly determined with the `Control` class. `HsmSourceMomentsRoundAlgorithm` is defined in this ticket, and is subclassed solely to have a different docstring.